### ISSUE TITLE
Remote Access Issue

### ISSUE THREAD

**Issue by RajarupanSampanthan** [dilevin/computer-graphics-shader-pipeline#29]
I'm trying to run the very first command given in the README.md   : ./shaderpipeline ../data/test-01.json. However I am unable to open a window. I see this error : 

![Screenshot from 2020-02-16 17-10-12](https://user-images.githubusercontent.com/18388850/74613795-940c8900-50df-11ea-9c38-c1dd330dd84d.png)

This is the first time I've ran into issues with remote access, so I'm a bit stumped into what could be the issue.  I've tried ssh commands  in the form :  ssh username@teach.cs.utoronto.ca -X and ssh username@teach.cs.utoronto.ca -Y and they give the same error. Anyone else run into this issue?

**Comment by dilevin** [comment]
Remote access to CDF is hit or miss. When you ssh in you have no guarantee you will get a machine with appropriate OpenGL hardware and software. This [issue](https://github.com/dilevin/computer-graphics-shader-pipeline/issues/12) sums it all up.

TLDR; remote access via CDF, for openGL applications is annoying,
